### PR TITLE
Update default nginx client_max_body_size configuration

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
Plugins are generally larger than 50mb

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by agnivade — APPROVED** [review]
Curious to know how you ran into this? I thought plugins were uploaded via mmctl?

**Comment by isacikgoz** [comment]
> Curious to know how you ran into this? I thought plugins were uploaded via mmctl?

Isn't `mmctl` hitting the same route over http? I didn't bother using local socket in HA mode.

File: server/build/docker/nginx/default.conf [hunk 1]
```diff
@@ -24,7 +24,7 @@ server {
   }
 
   location / {
-    client_max_body_size 50M;
+    client_max_body_size 100M;
```
Comment (streamer45) [inline server/build/docker/nginx/default.conf:27]: Going one step further, should we perhaps consider updating [our docs](https://docs.mattermost.com/install/config-proxy-nginx.html)? I am thinking for consistency more than anything, considering our config default max file size [is actually 100MB](https://github.com/mattermost/mattermost/blob/ed58b10486a1325c473d31d7aeafdebe712f4d00/server/public/model/config.go#L1614) so we are literally recommending a potentially broken config.

Comment (isacikgoz) [reply server/build/docker/nginx/default.conf:27]: Makes sense, I'll create a PR for that if we have a consensus on 100mb :)

**Review by streamer45 — APPROVED** [review]

**Pull request merged**